src/main/scala/Reader.scala

```diff
@@ -36,12 +36,30 @@ case object SpecFormatYAML extends SpecFormat
 
 class Reader{
   def readFile (fileName:String) = {
-    val content = Source.fromURL(getClass.getResource(fileName)).getLines().mkString("\n")
+    Source.fromURL(getClass.getResource(fileName)).getLines().mkString("\n")
+  }
+
+  def readWebUrl (WbUrl:String) = {
+    Source.fromURL(WbUrl).mkString
+  }
+
+  def specType (fileName:String) = {
     fileName match {
-      case file if file.endsWith("json") => (SpecFormatJSON, content)
-      case file if file.endsWith("yaml") => (SpecFormatYAML, content)
+      case file if file.endsWith("json") => (SpecFormatJSON)
```

Comment:
This is still being returned as tuple. Remove the paranthesis and return SpecFormat directly